src/components/app.tsx

```diff
@@ -31,6 +31,8 @@ import "./app.scss";
 
 const kDefaultActivity = "sample-activity-multiple-layout-types";   // may eventually want to get rid of this
 const kDefaultIncompleteMessage = "Please submit an answer first.";
+// TODO: switch default to production report version before production deploy
+export const DEFAULT_PORTAL_REPORT_URL = "https://portal-report.concord.org/branch/master/";
```

Comment:
I would probably add this to a utils module.  Seems a little odd to have it here.

Reply:
Also the AP tends to use kConstant over CONSTANT notation.